prepare.sh

```diff
@@ -0,0 +1,11 @@
+# move mathjax to static folder for our hacky loading.
```

Comment:
this script is not used for update the Perseus version. Perseus is pinned, and manually updated.